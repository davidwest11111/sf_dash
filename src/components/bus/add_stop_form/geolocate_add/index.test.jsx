import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import GeolocateAdd from './index';


describe('outputs the expected tree when', () => {
  let wrapper;

  test('(default)', () => {
    const addForm = {
      getNearbyStops: () => {},
      addStop: () => {},
      loading: false,
      errMsg: '',
      nearbyStops: [
        {
          tag: 'FIXME',
          stopTag: 'FIXME',
          title: 'FIXME',
          directionTitle: 'FIXME',
          stopTitle: 'FIXME',
        },
      ],
    };
    wrapper = shallow((
      <GeolocateAdd addForm={addForm} />
    ));
  });

  afterEach(() => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
