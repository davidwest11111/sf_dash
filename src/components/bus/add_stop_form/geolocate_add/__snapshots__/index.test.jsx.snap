// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when (default) 1`] = `
<Fragment>
  <Alert
    closeAriaLabel="Close"
    color="danger"
    fade={true}
    isOpen={true}
    tag="div"
    transition={
      Object {
        "appear": true,
        "baseClass": "fade",
        "baseClassActive": "show",
        "enter": true,
        "exit": true,
        "in": true,
        "mountOnEnter": false,
        "onEnter": [Function],
        "onEntered": [Function],
        "onEntering": [Function],
        "onExit": [Function],
        "onExited": [Function],
        "onExiting": [Function],
        "tag": "div",
        "timeout": 150,
        "unmountOnExit": true,
      }
    }
  >
    Warning: This probably 
    won't
     work. See
     
    <a
      href="https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only"
      rel="noopener noreferrer"
      target="_blank"
    >
      Geolocation API Removed from Unsecured Origins in Chrome 50
    </a>
    .
  </Alert>
  <Form
    className=""
    inline={true}
    tag="form"
  >
    <Label
      className="mr-2"
      tag="label"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      Find stops within
    </Label>
    <Input
      onChange={[Function]}
      type="select"
    >
      <option
        value=""
      >
        --Choose One--
      </option>
      <option
        value="322"
      >
        1/5 mile
      </option>
      <option
        value="402"
      >
        1/4 mile
      </option>
      <option
        value="804"
      >
        1/2 mile
      </option>
    </Input>
  </Form>
  <Table
    hover={true}
    responsiveTag="div"
    striped={true}
    tag="table"
  >
    <thead>
      <tr>
        <th>
          Route
        </th>
        <th>
          Direction
        </th>
        <th>
          Stop
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        key="FIXME|FIXME"
      >
        <td>
          FIXME
        </td>
        <td>
          FIXME
        </td>
        <td>
          FIXME
        </td>
      </tr>
    </tbody>
  </Table>
</Fragment>
`;
